3.2/root/usr/bin/usage

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cat /usr/share/container-scripts/redis/README.md
```

Comment:
Some other images uses  `groff` is usage script

https://github.com/sclorg/mariadb-container/blob/master/s2i-common/bin/usage

Reply:
Yes, although I'm not sure it should be a preferred way -- that would mean installing at least `groff-base` package more, and then showing the man page file is basically converting from markdown, to groff, then render it to plain text... which might or might not work as expected. Thinking about it more, simply showing README.md might be safer way..

Reply:
Make sense...